env.sh

```diff
@@ -58,6 +58,8 @@ export CARLA_MAP_PATH=${AUTOWARE_CARLA_ROOT}/carla_map/${CARLA_MAP_NAME}
 export ROS_DOMAIN_ID=0
 export VEHICLE_NAME="v1"
 
+# Set the ccache directory to /tmp
```

Comment:
Could you please add some comments here about the reason we need to set ccache to `/tmp`?